docs/unmanaged_configuration.md

```diff
@@ -25,3 +25,50 @@ oc set env dc/fluentd MERGE_JSON_LOG=true
 * Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings
 * Potential buffer storage leak caused by rejected message cycling
 * Overwrite of data for field with same names
+
+## ElasticSearch
+### openshift_logging_es_allow_external
+If openshift_logging_es_allow_external is set to True, Elasticsearch was exposed as a Route.
+The following steps will achieve the goal.
+* Create a route for ElasticSearch.
+```
+$ oc create -f - << EOF
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: elasticsearch
+  namespace: openshift-logging
+spec:
+  host: elasticsearch.openshift-logging.svc.cluster.local
+  to:
+    kind: Service
+    name: elasticsearch
+  tls:
+    termination: passthrough
```

Comment:
This isn't exactly the same as https://github.com/openshift/openshift-ansible/blob/5032be34a3e7746a77e19c561b17198aaae3148e/roles/openshift_logging_elasticsearch/tasks/main.yaml#L508 which creates a reencrypt route.  https://github.com/openshift/openshift-ansible/blob/5032be34a3e7746a77e19c561b17198aaae3148e/roles/openshift_logging_elasticsearch/templates/route_reencrypt.j2

Reply:
Yes, I also noticed kibana route has termination: reencrypt/Redirect...
Let me try replacing passthrough with it.